Content(missions): Hai Legacy (#8828)
endless-sky/endless-sky: data/hai/hai ships.txt

	attributes
		category "Medium Warship"
		"cost" 7400000
		"shields" 14200
		"hull" 6900
		"required crew" 20
		"bunks" 26
		"mass" 440
		"drag" 6
		"heat dissipation" .75
		"fuel capacity" 400
		"cargo space" 121
		"outfit space" 533
		"weapon capacity" 192
		"engine capacity" 93
		# Disabled faster due to the ship's sea time, which the Unfettered know about
		"threshold percentage" 0.4
		weapon
			"blast radius" 84
			"shield damage" 840
			"hull damage" 420
			"hit force" 1260
	outfits
		"Korath Detainer" 2
		"Bullfrog Anti-Missile" 4
		"Hai Williwaw Cooling"
		"Boulder Reactor"
		"Value Detector"
		"Asteroid Scanner"
		"Hai Valley Batteries"
		"Hai Diamond Regenerator"
		`"Biroo" Atomic Thruster`
		`"Bondir" Atomic Steering`
		"Hyperdrive"
		"Pulse Rifle" 20

	gun -24 -89.5
	gun 24 -89.5
	turret -40 -28
	turret 40 -28
	turret 0 60
	turret 0 -32.5
	engine -50 15
	engine 50 15
	"steering engine" -50 15
		zoom 0.5
		right
	"steering engine" 50.5 15
		zoom 0.5
		left
	leak "leak" 40 50
	leak "flame" 40 30
	explode "tiny explosion" 24
	explode "small explosion" 50
	explode "medium explosion" 30
	explode "large explosion" 14
	"final explode" "final explosion medium"

> 	description "First created during the Hai's initial conflicts with the Sheragi, the Anomalocaris was supposedly one of the most agile Hai warships of its time. Its high cargo capacity and low engine space suggest the first designs of the ship were intended for mining or freighting, but this also made it useful for looting enemy ships and carrying ammunition. Vessels from that time used a more expensive but also more durable material, allowing this ship to take quite a pounding, although its time under water may have damaged it."

Author: ```suggestion
	description "First created during the Hai's initial conflicts with the Sheragi, the Anomalocaris was supposedly one of the most agile Hai warships of its time. Its high cargo capacity and low engine space suggest the first designs of the ship were intended for mining or freighting, but this also made it useful for looting enemy ships and carrying ammunition. Vessels from that time used a more expensive material, allowing this ship to take quite a pounding, especially from big impacts, although its time under water may have damaged it."
```
lore update suggested by MasterOfGrey

	attributes
		category "Medium Warship"
		"cost" 7400000
		"shields" 14200
		"hull" 6900
		"required crew" 20
		"bunks" 26
		"mass" 440
		"drag" 6
		"heat dissipation" .75
		"fuel capacity" 400
		"cargo space" 121
		"outfit space" 533
		"weapon capacity" 192
		"engine capacity" 93
		# Disabled faster due to the ship's sea time, which the Unfettered know about
		"threshold percentage" 0.4
		weapon
			"blast radius" 84
			"shield damage" 840
			"hull damage" 420
			"hit force" 1260
	outfits
		"Korath Detainer" 2
		"Bullfrog Anti-Missile" 4
		"Hai Williwaw Cooling"
		"Boulder Reactor"
		"Value Detector"
		"Asteroid Scanner"
		"Hai Valley Batteries"
		"Hai Diamond Regenerator"
		`"Biroo" Atomic Thruster`
		`"Bondir" Atomic Steering`
		"Hyperdrive"
		"Pulse Rifle" 20

	gun -24 -89.5
	gun 24 -89.5
	turret -24 -89.5
	turret 24 -89.5
	turret 0 60
	turret 0.5 -32.5
	engine -50 15
	engine 50 15
	"steering engine" -50 15
		zoom 0.5
		right
	"steering engine" 50.5 15
		zoom 0.5
		left
	leak "leak" 40 50
	leak "flame" 40 30
	explode "tiny explosion" 24
	explode "small explosion" 50
	explode "medium explosion" 30
	explode "large explosion" 14
	"final explode" "final explosion medium"

> 	description "This ship is rumored to be one of the first agile warships the Hai made, when conflicts started to erupt with the Sheragi. The high cargo capacity and low engine space was probably due to the first designs of the ship being intended for mining or freighting instead, but it was useful to collect loot from the enemy ships and carry ammo as well as escorting more vulnerable ships. Vessels from that time used a more expensive but also more durable material, allowing this ship to take quite a pounding, although its time under water may have damaged it."

Reviewer: ```suggestion
	description "First created during the Hai's initial conflicts with the Sheragi, the Anomalocaris was supposedly one of the most agile warships of its time. Its high cargo capacity and low engine space suggest the first designs of the ship were intended for mining or freighting, but this also made it useful for collecting loot from enemy ships and carrying ammunition. Vessels from that time used a more expensive but also more durable material, allowing this ship to take quite a pounding, although its time under water may have damaged it."
```

Author: idk if I'd say "of it's time" bc it's not that agile and they quickly made more focused agile warships I imagine - such as the lightning bug, for instance

Reviewer: ```suggestion
	description "First created during the Hai's initial conflicts with the Sheragi, the Anomalocaris was supposedly one of the most agile Hai warships of its time. Its high cargo capacity and low engine space suggest the first designs of the ship were intended for mining or freighting, but this also made it useful for collecting loot from enemy ships and carrying ammunition. Vessels from that time used a more expensive but also more durable material, allowing this ship to take quite a pounding, although its time under water may have damaged it."
```
How about this?

Author: collecting loot could be shortened to looting no?

# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

ship "Anomalocaris"
	plural "Anomalocarises"
	sprite "ship/hai anomalocaris"
	thumbnail "thumbnail/hai anomalocaris"
	attributes
		category "Medium Warship"
		"cost" 7400000
		"shields" 14200
		"hull" 6800
		"required crew" 20
		"bunks" 26

> 		"mass" 440
> 		"drag" 6
> 		"heat dissipation" .65

Reviewer 2: ```suggestion
		"mass" 970
		"drag" 9.3
		"heat dissipation" .49
```
Mass rebalance changes

# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

ship "Anomalocaris"
	plural "Anomalocarises"
	sprite "ship/hai anomalocaris"
	thumbnail "thumbnail/hai anomalocaris"
	attributes
		category "Medium Warship"
		"cost" 7400000
		"shields" 14200
		"hull" 6800
		"required crew" 20
		"bunks" 26
		"mass" 970
		"drag" 9.3
		"heat dissipation" .49
		"fuel capacity" 400
		"cargo space" 121
		"outfit space" 533
		"weapon capacity" 192
		"engine capacity" 98
		weapon
			"blast radius" 210
			"shield damage" 2100
			"hull damage" 1050
			"hit force" 3150
	outfits
		`"Biroo" Atomic Thruster`
		`"Bondir" Atomic Steering`
		"Boulder Reactor"
		"Chameleon Anti-Missile" 2
		"Hai Diamond Regenerator"
		"Hai Ravine Batteries"
		"Hai Williwaw Cooling" 3
		Hyperdrive
		"Ion Cannon"
		"Pebble Core"
		"Pulse Cannon"
		"Pulse Rifle" 20
		"Pulse Turret" 2
		"Quantum Keystone"
	gun -24 -89.5
	gun 24 -89.5
	turret -40 -28
	turret 40 -28
	turret 0 60
	turret 0 -32.5
	engine -50 15

> 	engine 50 15
> 	"steering engine" -50 15
> 		zoom 0.5
> 		right
> 	"steering engine" 50.5 15
> 		zoom 0.5
> 		left
> 	leak "leak" 40 50

Reviewer 3: ```suggestion
	engine 50 15
	leak "leak" 40 50
```
The steering engines being the same engines, in this particular case, don't really look that good with the ship's mass center especially when turning without forward thrust. It looks less awkward to not have them here, or add small tiny ones to more appropriate places later.

	leak "leak" 40 50
	leak "flame" 40 30
	explode "tiny explosion" 24
	explode "small explosion" 50
	explode "medium explosion" 30
	explode "large explosion" 14
	"final explode" "final explosion medium"

> 	description "First created during the Hai's initial conflicts with the Sheragi, the Anomalocaris was supposedly one of the most agile Hai warships of its time. Its high cargo capacity and low engine space suggest the first designs of the ship were intended for mining or freighting, but this also made it useful for looting enemy ships and carrying ammunition. Vessels from that time used a more expensive material, allowing this ship to take quite a pounding, especially from big impacts."

Reviewer 4: ```suggestion
	description "First created during the Hai's initial conflicts with the Sheragi, the Anomalocaris was supposedly one of the most agile Hai warships of its time. Its high cargo capacity and low engine space suggest the first designs of the ship were intended for mining or freighting, but also made it useful for looting enemy ships and carrying ammunition. Vessels from that time used a more expensive material, allowing this ship to take quite a pounding, especially from larger impacts."
```
Some minor wording tweaks.

ship "Anomalocaris" "Anomalocaris (Venerable)"
	add attributes
		# Disabled faster due to the ship's sea time, which the Unfettered know about
		"threshold percentage" 0.4
	outfits
		"Korath Detainer" 2
		"Bullfrog Anti-Missile" 4
		"Hai Williwaw Cooling"
		"Boulder Reactor"
		"Value Detector"
		"Asteroid Scanner"
		"Hai Valley Batteries"
		"Hai Diamond Regenerator"
		`"Biroo" Atomic Thruster`
		`"Bondir" Atomic Steering`
		"Hyperdrive"
		"Pulse Rifle" 20

> 	description "First created during the Hai's initial conflicts with the Sheragi, the Anomalocaris was supposedly one of the most agile Hai warships of its time. Its high cargo capacity and low engine space suggest the first designs of the ship were intended for mining or freighting, but this also made it useful for looting enemy ships and carrying ammunition. Vessels from that time used a more expensive material, allowing this ship to take quite a pounding, especially from big impacts, although its time under water may have damaged it."

Reviewer 4: ```suggestion
	description "First created during the Hai's initial conflicts with the Sheragi, the Anomalocaris was supposedly one of the most agile Hai warships of its time. Its high cargo capacity and low engine space suggest the first designs of the ship were intended for mining or freighting, but also made it useful for looting enemy ships and carrying ammunition. Vessels from that time used a more expensive material, allowing this ship to take quite a pounding, especially from larger impacts, although its time under water may have damaged it."
```

		`"Biroo" Atomic Steering`
		"Hyperdrive"

> ship "Lightning Bug" "Lighting Bug (Tripulse)"

Author: ```suggestion
ship "Lightning Bug" "Lightning Bug (Tripulse)"
```

Reviewer 5: Can't you add these changes yourself? This is just a typo.

		`"Biroo" Atomic Steering`
		"Hyperdrive"

# this design purposely lacks energy
ship "Lightning Bug" "Lightning Bug (Ionic)"
	outfits
		"Ionic Blaster Prototype"
		"Ionic Turret Prototype"
		"Hai Corundum Regenerator"
		"Hai Williwaw Cooling"
		"Quantum Keystone"
		"Value Detector"
		"Sand Cell" 3
		"Hai Fissure Batteries"
		"Pulse Rifle" 10
		`"Benga" Atomic Thruster`
		`"Biroo" Atomic Steering`

> 		"Hyperdrive"

Reviewer 5: I just noticed the other Lightning Bug variants were in alphabetical order.

	"final explode" "final explosion small"
	description "The Lightning Bug is a small defensive ship, mostly used by the Hai as a convoy escort. The Lightning Bug, as with many insects, works best in a swarm: even a small number of Lightning Bugs can manage to leave the largest of ships helpless with their stock-fitted Ion Cannons."

> ship "Lightning Bug" "Lightning Bug (Ionic)"

Author: There's 1 turret but I see what you mean

I dont think its necessary to fill all hardpoints, and the sea scorp isnt a garanteed spawn iirc
This turret is in fact the biggest threat in this fight

> ship "Lightning Bug" "Lightning Bug (Ionic)"

Reviewer 3: Lightning Bugs have one gun mount and two turret mounts.

> ship "Lightning Bug" "Lightning Bug (Ionic)"

Author: I added a bullfrog to it instead of removing the turret as I feel like a lightning bug with just a blaster (which is just a glorified pulse cannon, at 19 space) isn't that relevant in battle, and I want this variant to be more generic than just for this encounter.
The point could be made to make it weak if the fight is estimated to be too hard, of course

	"final explode" "final explosion small"
	description "The Lightning Bug is a small defensive ship, mostly used by the Hai as a convoy escort. The Lightning Bug, as with many insects, works best in a swarm: even a small number of Lightning Bugs can manage to leave the largest of ships helpless with their stock-fitted Ion Cannons."

# this design purposely lacks energy
ship "Lightning Bug" "Lightning Bug (Ionic)"
	outfits
		"Ionic Blaster Prototype"
		"Ionic Turret Prototype"
		"Hai Corundum Regenerator"
		"Hai Williwaw Cooling"
		"Quantum Keystone"
		"Value Detector"
		"Sand Cell" 3
		"Hai Fissure Batteries"
		"Pulse Rifle" 10
		`"Benga" Atomic Thruster`
		`"Biroo" Atomic Steering`

> 		"Hyperdrive"

Author: well even after the fact it'd still be vulnerable to hai ion cannons I think

	"final explode" "final explosion small"
	description "The Lightning Bug is a small defensive ship, mostly used by the Hai as a convoy escort. The Lightning Bug, as with many insects, works best in a swarm: even a small number of Lightning Bugs can manage to leave the largest of ships helpless with their stock-fitted Ion Cannons."

> 
> ship "Lightning Bug" "Lightning Bug (Ionic)"
> 	outfits
> 		"Ionic Blaster Prototype"
> 		"Ionic Turret Prototype"
> 		"Bullfrog Anti-Missile"
> 		"Hai Corundum Regenerator"
> 		"Hai Cuttlefish Jammer"
> 		"Outfits Expansion"
> 		"Value Detector"
> 		"Hai Gorge Batteries"
> 		"Hai Williwaw Cooling"
> 		"Quantum Keystone"
> 		Hyperdrive
> 		"Geode Reactor"
> 		`"Benga" Atomic Thruster`
> 		`"Biroo" Atomic Steering`
> 		"Pulse Rifle" 10
> 

Author: ```suggestion
```

		"Ion Cannon"
		"Ionic Blaster Prototype" 2
		"Ionic Turret Prototype"

> 		"Pulse Rifle" 36

Reviewer 5: What is this about?

Author: oh I was originally using it but then not and I didnt cancel the change
it's about prototype ships having good defense

Reviewer 5: Ah, OK.

Author: ```suggestion
		"Pulse Rifle" 15
```
it shouldnt be changed here tho you're right

		"Hai Williwaw Cooling" 4
		"Hyperdrive"
		"Pulse Cannon" 2

> 		"Pulse Rifle" 36

Reviewer 5: Is this necessary? You can't complete the mission if you capture this ship.

Author: you can capture every single one of these ships and complete the mission, all that is required is to disable, not to save
this is not a standard duel, as the UHai warn in the convo
also yes it is very much necessary to show they care about the prototypes

Reviewer 5: `npc disable` means that you have to leave the npcs disabled to complete the mission.

Author: npc disable means you must have disabled them at one point, and even if they get repaired afterwards it's still valid

Reviewer 5: OK, thanks.

		"Ion Cannon"
		"Ionic Blaster Prototype" 2
		"Ionic Turret Prototype"

> 		"Pulse Rifle" 36

Author: yeah caus it will actually spawn here

> 		"Pulse Rifle" 36

Reviewer 6: Well the comment just above mine (on the same line) you said that you were going to change it back.

> 		"Pulse Rifle" 36

Author: I changed my mind about changing my mind since x)

ship "Sea Scorpion" "Sea Scorpion (Shredder)"

> 	crew 30

Author: ```suggestion
```

		"Hyperdrive"
		"Pulse Rifle" 15

# These are prototype equipped ship that do not spawn normally, as such they are guarded against boarding,

> # although part of the crew on the ship are engineers monitering the success of the weapons.

Reviewer 4: ```suggestion
# although part of the crew on the ship are engineers monitoring the success of the weapons.
```

		"Hai Williwaw Cooling" 4
		"Hyperdrive"
		"Pulse Cannon" 2

> 		"Pulse Rifle" 36

Author: ```suggestion
		"Pulse Rifle" 15
```

		`"Bufaer" Atomic Steering`
		"Hyperdrive"

> ship "Shield Beetle" "Shield Beetle (Prototypes)"

Reviewer 3: ```suggestion
ship "Shield Beetle" "Shield Beetle (Prototype)"
```

		`"Bufaer" Atomic Steering`
		"Hyperdrive"

> ship "Shield Beetle" "Shield Beetle (Tripulse Ionic Turret)"
> 	crew 80
> 	outfits
> 		"Pulse Cannon" 3
> 		"Tripulse Shredder" 3
> 		"Hai Tracker" 112
> 		"Hai Tracker Pod" 2
> 		"Ionic Turret Prototype"
> 		"Chameleon Anti-Missile" 3

Reviewer 3: ```suggestion
ship "Shield Beetle" "Shield Beetle (Tripulse Ionic Turret)"
	crew 80
	outfits
		"Ion Cannon" 2
		"Pulse Cannon" 2
		"Pulse Turret"
		"Tripulse Shredder" 2
		"Hai Tracker" 112
		"Hai Tracker Pod" 2
		"Ionic Turret Prototype"
		"Chameleon Anti-Missile" 2
		"Bullfrog Anti-Missile"
```

Author: you didnt remove the ionic turret haha
```suggestion
ship "Shield Beetle" "Shield Beetle (Tripulse Ionic Turret)"
	crew 80
	outfits
		"Ion Cannon" 2
		"Pulse Cannon" 2
		"Pulse Turret"
		"Tripulse Shredder" 2
		"Hai Tracker" 112
		"Hai Tracker Pod" 2
		"Chameleon Anti-Missile" 2
		"Bullfrog Anti-Missile"
```
